src/main/kotlin/edu/usf/cutr/otp/planner/model/core/OptimzeType.kt

```diff
@@ -0,0 +1,6 @@
+package edu.usf.cutr.otp.planner.model.core
+
+enum class OptimizeType {
```

Comment:
There isn't a direct match in OTP2 for this, but I think these enum values should cover most of the responses:
https://github.com/opentripplanner/OpenTripPlanner/blob/4c22825aef84619e1ac3792ede23555ed6f1f91d/src/main/java/org/opentripplanner/routing/core/BicycleOptimizeType.java